fire/cli/ts/_plot_gnss.py

```diff
@@ -0,0 +1,283 @@
+from typing import Callable
+
+import matplotlib.pyplot as plt
+from matplotlib.ticker import FormatStrFormatter
+import numpy as np
+
+from fire.api.model import GNSSTidsserie
+
+from fire.api.model.tidsserier import PolynomieRegression1D
+
+
+GNSS_TS_PLOTTING_LABELS = {
+    "t": "Dato",
+    "x": "x [mm]",
+    "y": "y [mm]",
+    "z": "z [mm]",
+    "X": "X [mm]",
+    "Y": "Y [mm]",
+    "Z": "Z [mm]",
+    "n": "Nord [mm]",
+    "e": "Øst [mm]",
+    "u": "Op [mm]",
+    "decimalår": "År",
+}
+
+
+def plot_gnss_ts(
+    ts: GNSSTidsserie, plot_funktion: Callable, parametre: list = ["n", "e", "u"]
+):
+    """
+    Plotter en GNSSTidsserie.
+
+    Denne funktion håndterer figuropsætningen, og kalder ``plot_funktion``, som
+    forventes at foretage selve plottingen af data.
+    """
+    # Skalér y-værdien for at vise data i mm
+    skalafaktor = 1e3
+
+    n_parm = min(len(parametre), 3)
+
+    ax = plt.figure()
+    plt.suptitle(ts.navn)
+    plt.xlabel("År")
+
+    for i, parm in enumerate(parametre, start=1):
+        if i > 3:
+            break
+
+        # BUG: Næste linje fejler hvis yy er None. Sker fx hvis man vælger ts-parametrene
+        # koordinatkovarians og residualkovarians.
+        y = [skalafaktor * yy for yy in getattr(ts, parm)]
+
+        try:
+            label = GNSS_TS_PLOTTING_LABELS[parm]
+        except KeyError:
+            label = parm
```

Comment:
Véd ikke, om den forbedrer performance, men her er et alternativ, som jeg foretrækker:

```suggestion
    if parm in GNSS_TS_PLOTTING_LABELS:
        label = GNSS_TS_PLOTTING_LABELS[parm]
    else:
        label = parm
```

Reply:
Hej Joachim
Ja der er de to måder at gøre det på, og de gør begge det samme. Performance er ikke et problem her (dog vil jeg tro at try... except er hurtigere.. ) og jeg foretrækker den her måde, så beholder det bare som det er :-)